src/Components/Artwork/GridItem.tsx

```diff
@@ -167,7 +193,11 @@ class ArtworkGridItemContainer extends React.Component<Props, State> {
               }
             }}
           >
-            <Image src={this.getImageUrl()} />
+            <Image
+              style={{ opacity: isImageLoaded ? "1" : "0" }}
+              src={this.getImageUrl()}
+              onLoad={this.imageLoaded.bind(this)}
+            />
```

Comment:
Will there be a `img` tag with `src` attribute coming from the SSR content? If not, does that affect SEO in any way?

Reply:
I had the same question actually... will point nicolette to this for her thoughts.

Reply:
From an SSR perspective, no. Based on google’s own recommendations, this should be fine as long as the image does indeed pull in as the bot scrolls. https://developers.google.com/search/docs/guides/lazy-loading. Traditionally lazy-loaded imgs use the `data-src` attribute as a placeholder before loading, but that’s not the case with this library.

Reply:
Would just raise that traditionally `src` and `alt` are used to help Google understand what the image is. Since `src` is typically a cloudfront URL for us without a description of the artwork I think it's okay to forego that, however, can we add `alt` text to the images? They could just be the artwork title. 

I believe we used to have these on our artist pages, or at least, in the snapshot we provided to Google through reflection.

Reply:
I actually noticed we're not really doing a good job of that anymore... I'll try to fix it up. 👍

Reply:
I've buttoned this up. `title` attributes come from the artwork `title` field and the `alt` attribute is provided by the `image_title` field (that has details such as author and year).